### PR TITLE
FIrst commit to test custom env

### PR DESCRIPTION
**Pull request by bame-da**
This is a copy of my mockup scenario together with a skeletal custom environment to get the ball rolling on the custom setup side.

### CODE REVIEW

File: test/step1.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Once the prompt says "Ready to go", you can start using the DAML IDE. Use VSCode in the IDE Tab.
+You can run `daml start`{{execute}} to run your project and open the navigator (or any other UI) by clicking the "+" symbol next to he IDE tab. Navigator runs on [port `7500`](https://[[HOST_SUBDOMAIN]]-7500-[[KATACODA_HOST]].environments.katacoda.com/) by default.
```
Comment (cocreature) [inline test/step1.md]:
```suggestion
You can run `daml start`{{execute}} to run your project and open the navigator (or any other UI) by clicking the "+" symbol next to the IDE tab. Navigator runs on [port `7500`](https://[[HOST_SUBDOMAIN]]-7500-[[KATACODA_HOST]].environments.katacoda.com/) by default.
```

File: test/step2.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+## Quizzes
```
Comment (cocreature) [inline test/step2.md]: Can we just kill this file?

**Review by cocreature — APPROVED** [review]
LGTM, thanks!

File: test/step2.md (hunk 2 above)
Comment (bame-da) [reply test/step2.md]: Sure, tidied away all content.

**Pull request merged**